### ISSUE TITLE
This project seems to be giving errors on npm install.

### ISSUE THREAD

**Issue by skozzii** [kevinpiac/vue2-firebase-boilerplate#1]
Looks to be out of date and giving errors.  Just a heads up for others.

Looks like a good package though.

**Comment by AsenCME** [comment]
Yes, the error says that it's missing css/stylus/etc. loaders and also babel, for some reason.

**Comment by kevinpiac** [comment]
Sorry guys, this boilerplate has not been updated for a while and I don't plan to maintain it since projects like Nuxt are doing a really great job! Why don't using Nuxt directly?

**Comment by kevinpiac** [comment]
By the way it seems that the package.json does not list all dependencies. I've tried with a more recent version and it works. Unfortunately, I can't tell you precisely which dependencies are missing.
